upload_controller_service/core/main.py

```diff
@@ -136,6 +145,7 @@ def check_uploaded_file(
     with Database(config=config) as database:
         try:
             file = database.get_file(file_id=file_id)
+            database.update_file_state(file_id=file_id, state=UploadState.UPLOADED)
```

Comment:
Could you please move the object storage with block into the database with block and perform the state update after checking the file existence in the storage.